maat-orchestration/src/main/java/uk/gov/justice/laa/maat/orchestration/dto/HRDetailDescriptionDTO.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.justice.laa.maat.orchestration.dto;
+
+public class HRDetailDescriptionDTO extends GenericDTO {
+
+    private String code;
+    private String description;
+
+    @Override
+    public Object getKey() {

```

Comment:
If this DTO doesn't have a key defined it shouldn't extend GenericDTO, there are a few different approaches which can be taken here without the fulkl context I'm not sure which to suggest.